.github/workflows/ci.yml

```diff
@@ -25,19 +25,18 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: 3.7.4
+          mamba-version: "0.16.0"
+          channels: bioconda,conda-forge,defaults
+          channel-priority: true
           auto-update-conda: false
-          environment-file: install/environment.root.yml
           activate-environment: zarp
+          environment-file: install/environment.root.yml
```

Comment:
root env can't be used directly, instead the general env has to be installed first and then updated with root?    
Check whether github "setup miniconda" action allows that. (Probably yes)